webpack.prod.config.js

```diff
@@ -0,0 +1,14 @@
+var path = require('path');
+var merge = require('webpack-merge').merge;
+var commonConfiguration = require('./webpack.config.js');
+
+module.exports = merge(commonConfiguration, {
+  output: {
+    library: 'AFRAME',
+    libraryTarget: 'umd',
+    path: path.resolve(__dirname, 'dist'),
+    publicPath: '/dist/',
+    filename: 'aframe-master.min.js'
+  },
+  mode: 'production'
+});
```

Comment:
This file exists just to change the filename and the mode to production so that we have a minified build.
https://webpack.js.org/configuration/mode/